### ISSUE TITLE
Input mapping helper class

### ISSUE THREAD

**Issue by Gama11** [HaxeFlixel/flixel#752]
Something like `FlxInputMapping`, where you can define an input similar to this pseudo-code:

``` haxe
FlxInputMapping.add("Jump", [Array of allowed jump keys], [Array of allowed gamepad buttons], callbackFunction);

public function callbackFunction():Void
{
    // Jumping logic
}
```
##

**Comment by MSGhero** [comment]
Should Lars' recent set of commits close this? `FlxActionWhatever`?

**Comment by Gama11** [comment]
Closed by #1805.
